geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/AlterMappingFunction.java

```diff
@@ -84,4 +88,9 @@ private CliFunctionResult createSuccessResult(String connectionName, String memb
     String message = "Altered JDBC connection " + connectionName + " on " + member;
     return new CliFunctionResult(member, xmlEntity, message);
   }
+
+  @Override
+  public Collection<ResourcePermission> getRequiredPermissions(String regionName) {
```

Comment:
maybe for functions that "alters" something, it should be cluster:write?

Reply:
Alter connections/mappings does change `cluster-config` which is very similar to create & destroy operations.
Also, most other alter operations do use `X:MANAGE:X` permission.